src/app/ResourceManager/LocalizationHelpers.cs

```diff
@@ -80,24 +81,17 @@ public static string GetSubmoduleText(IGitModule superproject, string name, stri
             StringBuilder sb = new();
             sb.AppendLine("Submodule " + name);
             sb.AppendLine();
-            IGitModule module = superproject.GetSubmodule(name);
 
             // Submodule directory must exist to run commands, unknown otherwise
-            if (module.IsValidGitWorkingDir())
+            IGitModule module = superproject.GetSubmodule(name);
+            if (module.IsValidGitWorkingDir()
+                && new CommitDataManager(() => module).GetCommitData(hash) is CommitData data)
```

Comment:
❔ Never cache any longer?

Reply:
Using Git cache as before (due to #12470, commit info was not cached)